website/src/content/main-concepts/api-gateway.mdx

```diff
@@ -10,9 +10,61 @@ To solve these issues, the Merchant Center provides an **API Gateway**, which pr
 
 ![HTTP API](/images/mc-http-api.png)
 
-All Merchant Center applications should send requests through the API Gateway, which is available at the same domain of the Merchant Center but with the `mc` subdomain being `mc-api`.
+<Info>
 
-Official domains:
+It is recommended that Custom Applications use the API Gateway to access the commercetools APIs, for security reasons. See [Proxy Endpoints](/main-concepts/proxy-endpoints).
```

Comment:
```suggestion
For security reasons, it is recommended that Custom Applications use the API Gateway to access the commercetools APIs. See [Proxy Endpoints](/main-concepts/proxy-endpoints).
```

Reply:
@emmenko  this grammar change is correct and should be applied.   

Plus, I would not even use "recommended" but prescribe it in an "it's required to" so you have to chance to actually filter out direct API hosts in the API gateway CSP header generators.